docs/pages/installation/docker-extension.md

```diff
@@ -17,9 +17,15 @@ The Docker Extension for Meshery extends Docker Desktop’s position as the clou
 
 ### Using Docker Desktop
 
-Navigate to the Extensions area of Docker Desktop.
+1) Navigate to the Extensions area of Docker Desktop.
 
-### Using `docker`
+2) Then navigate to Meshery in the MarketPlace and press install.
+
+<a href="{{ site.baseurl }}/assets/img/platforms/docker-extensions-marketplace.png">
+  <img style="width:350px;" src="{{ site.baseurl }}/assets/img/platforms/docker-extensions-marketplace.png">
+</a>
+
```

Comment:
There is one more step to this, showing Meshery in the marketplace and using the `install` button.

Reply:
Thanks for the feedback , I would definately make those changes  ✨